cmd/dockerd/daemon.go

```diff
@@ -271,17 +271,19 @@ func newRouterOptions(config *config.Config, daemon *daemon.Daemon) (routerOptio
 	if err != nil {
 		return opts, err
 	}
-
-	buildkit, err := buildkit.New(buildkit.Opt{
-		SessionManager: sm,
-		Root:           filepath.Join(config.Root, "buildkit"),
-		Dist:           daemon.DistributionServices(),
-	})
-	if err != nil {
-		return opts, err
+	var builder *buildkit.Builder
```

Comment:
you can call the variable `bk` since this is specifically the buildkit builder. Otherwise it's weird to see `builder` being `nil` when buildkit is disabled.